hyperliquid/exchange_plus.py

```diff
@@ -0,0 +1,106 @@
+from eth_account.signers.local import LocalAccount
+
+from hyperliquid.info import Info
+from hyperliquid.exchange import Exchange
+from hyperliquid.utils.types import Any, List, Literal, Meta, Optional, Tuple, Cloid
+
+# Extends Exchange Class with 'Market Orders'
+class ExchangePlus(Exchange):
+    DEFAULT_SLIPPAGE = 0.05 # Max allowed Slippage for Market Orders
+
+    def __init__(
+        self,
+        wallet: LocalAccount,
+        base_url: Optional[str] = None,
+        meta: Optional[Meta] = None,
+        vault_address: Optional[str] = None,
+    ):
+        super().__init__(wallet, base_url, meta, vault_address) 
+        self.info = Info(base_url, skip_ws=True)
+
+        # create a szDecimals map
+        self.sz_decimals = {}
+        for asset_info in self.meta["universe"]:
+            self.sz_decimals[asset_info["name"]] = asset_info["szDecimals"]        
+
+    def __round_sz(
```

Comment:
Personally, I don't think it's a good idea to round the sizes for the user. Is this something that comes up often for you that rounding would be helpful for? Do other sdk's provide this feature? cc: @chameleon-jeff

Reply:
No, I don't think many sdk offer that. I found it useful myself as I could just pass my calculated order size into the market order. I can remove it, np. There is a rounding example in the sdk anyways.